lib/shape.ml

```diff
@@ -1,10 +1,11 @@
 open Graphics
 
 type point = { x : int; y : int }
+type color = RGB of int * int * int | Transparent
```

Comment:
Should we call it something like `Default` rather than `Transparent`? For stroke, this is associated with `black`.

Reply:
I'm not sure if I clearly understand. Did you mean that instead of using `Transparent` for the fill color, I rather use `Default`?

Reply:
Sorry, my wording wasn't precise. I meant renaming `Transparent` to `Default` so it's useful to both stroke and fill, where stroke defaults to black and fill defaults to transparent.

Reply:
> Sorry, my wording wasn't precise. I meant renaming `Transparent` to `Default` so it's useful to both stroke and fill, where stroke defaults to black and fill defaults to transparent.

I renamed it already. Please check my most recent commit